Vagrantfile

```diff
@@ -120,6 +120,16 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     vb.cpus = vm_num_cpus
   end
 
+  config.vm.provider "parallels" do |prl, override|
+	# We are using a ubuntu-20 box here since other
+	# boxes which support parallels are outdated and
+	# not popularly downloaded.
+	override.vm.box = "bento/ubuntu-20.04"
+	override.vm.box_version = "202005.21.0"
+	prl.memory = vm_memory
+	prl.cpus = vm_num_cpus
+  end
```

Comment:
The commit message should be about adding support for Parallels, since that's what this does, right?

Reply:
Yes.